### PR TITLE
fix: use primitive type multires example

### PR DESCRIPTION
**Pull request by JoanFM**

### CODE REVIEW

File: multires-lyrics-search/app.py [hunk 1]
```diff
@@ -30,7 +29,7 @@ def input_fn():
         reader = csv.reader(f)
         for row in itertools.islice(reader, int(os.environ['JINA_MAX_DOCS'])):
             if row[-1] == 'ENGLISH':
-                d = jina_pb2.Document()
+                d = Document()
```
Comment (cristianmtr) [inline multires-lyrics-search/app.py:32]: I think you need to do `update_hash` or use `Document` as context manager for it to be called automatically on `__exit__`

Comment (JoanFM) [reply multires-lyrics-search/app.py:32]: yes, but will it have any impact in this example?

Comment (JoanFM) [reply multires-lyrics-search/app.py]: Done!

File: multires-lyrics-search/app.py [hunk 2]
```diff
@@ -72,10 +71,10 @@ def main():
         workspace = os.environ['JINA_WORKSPACE']
         if os.path.exists(workspace):
             print(
-                f'\n +---------------------------------------------------------------------------------+ \
-                    \n |                                   🤖🤖🤖                                        | \
+                f'\n +-----------------------------------------------------------------------------------+   \
```
Comment (cristianmtr) [inline multires-lyrics-search/app.py:74]: Does this break the rendering?

Comment (JoanFM) [reply multires-lyrics-search/app.py:74]: oh yes, thanks!

**Review by cristianmtr — APPROVED** [review]

**Pull request merged**